force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -230,16 +330,23 @@ public with sharing class ClassesObjectsMethods {
         // title
         // author
         // pages
-    
+        public String title;
+        public String author;
+        public Integer pages;
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(string title, String author, Integer pages) 
```

Comment:
In this case, the comment is instructing you to include the title and author only in this constructor (but you added pages as well). This may be one reason you receive an error stating the class/method didn't exist. This is something to discuss in a 1-on-1 session.